lib/structures/Member.js

```diff
@@ -56,6 +61,8 @@ class Member extends Base {
         this.status = data.status !== undefined ? data.status : this.status || "offline";
         this.game = data.game !== undefined ? data.game : this.game || null;
         this.joinedAt = data.joined_at !== undefined ? Date.parse(data.joined_at) : this.joinedAt;
+        this.clientStatus = Object.assign({}, { web: "offline", desktop: "offline", mobile: "offline" }, data.client_status);
```

Comment:
What's the reason for the `{}`? Also this needs to handle client_status undefined/null, update docs accordingly.

Reply:
Initially, it was so it wouldn't modify the object containing the defaults, but I suppose that's not necessary. Will be changed.

Is there ever a case where it's `null`? From what I can tell, it's always defined, or if not, the property just isn't there.